src/utils.js

```diff
@@ -302,3 +306,20 @@ export function fuzzyMatch(keyword, candidate) {
 export function openSettings(tab) {
   routeTo({ path: '/settings', query: { tab } });
 }
+
+export function promptWhenGstUnavailable() {
```

Comment:
This function need not be a part of utils, it can move inside gst.js

Reply:
Done 💯